relaynet-cogrpc-jvm/src/main/java/tech/relaycorp/relaynet/cogrpc/client/CogRPCClient.kt

```diff
@@ -131,7 +144,8 @@ class CogRPCClient(
             AuthorizationMetadata.makeMetadata(cca)
         )
 
-    class Exception(throwable: Throwable) : kotlin.Exception(throwable)
+    class ServerException(throwable: Throwable) : Exception(throwable)
```

Comment:
Can you please call this class `CogRPCException`? `throwable` may not always originate on the server -- It may originate on the client too, when the server doesn't meet the deadline, for example.